src/internal/m365/backup.go

```diff
@@ -204,35 +203,81 @@ func verifyBackupInputs(sels selectors.Selector, siteIDs []string) error {
 	return nil
 }
 
-type getInfoer interface {
-	GetInfo(context.Context, string) (*api.UserInfo, error)
-}
-
 func checkServiceEnabled(
 	ctx context.Context,
-	gi getInfoer,
+	ctrl *Controller,
```

Comment:
This is a bit overbearing, since y7ou're still only using `ctrl.AC.foo()`.  I'd suggest passing along the api client instead of the full controller.  That way you can still use `ac.Users()` and `ac.Sites()` as needed.
```suggestion
	ac api.Client,
```

Reply:
Yeah, that's fair.